storage/storage-client/src/lib.rs

```diff
@@ -635,6 +639,27 @@ impl DbReader for SyncStorageClient {
                 .spawn(async move { reader.get_latest_state_root().await }),
         )?
     }
+
+    fn update_to_latest_ledger(
```

Comment:
probably something to follow up next, but since we are gonna get off the protobuf based API, shall we convert the call sites to use APIs provided by Json-RPC (hence DbReader) and deprecate `update_to_latest_ledger()`?

Reply:
@christinacelee To confirm, am I right that we are not gonna need `update_to_latest_ledger()` in JSON-RPC (hence everywhere)?

Reply:
not in the current implementation, but @JoshLind might make some changes that need them?

Reply:
Thanks @christinacelee. Matches my understanding. I think in that case we should hesitate to add it to `DbReader` while migrating off `StorageRead` -- otherwise we are just creating tech debt, I'd convert call sites to finer grained APIs like `.get_account_state()` etc as we go.

Reply:
cool. Then I will go with @msmouse